src/content/docs/en/guides/integrations-guide/markdoc.mdx

```diff
@@ -75,18 +75,29 @@ export default defineConfig({
 });
 ```
 
-### Editor Integration
+### VS Code Editor Integration
```

Comment:
Also note to translators: This new section is a combination of the older Editor Integration and #Markdoc langauge server. They have both been combined into this one!